src/bin/pgcopydb/catalog.c

```diff
@@ -143,7 +143,9 @@ static char *sourceDBcreateDDLs[] = {
 	"create table sentinel("
 	"  id integer primary key check (id = 1), "
 	"  startpos pg_lsn, endpos pg_lsn, apply bool, "
-	" write_lsn pg_lsn, flush_lsn pg_lsn, replay_lsn pg_lsn)"
+	" write_lsn pg_lsn, flush_lsn pg_lsn, replay_lsn pg_lsn)",
+
+	"create table lsn_tracking(source pg_lsn, target pg_lsn)"
```

Comment:
Are you planning to move file based lsn tracking to sqlite? 

(I don't see this table being used anywhere.)

Reply:
Ooops, missed the clean-up for that. The idea crossed my mind yeah, but I wanted to clean-up some other aspects first (summary information and top-level timings in the SQLite database so that `pgcopydb list summary` can represent multi-steps activity and also be used concurrently to the main command, or after the fact).